core/src/main/java/org/apache/carbondata/core/metadata/SegmentFileStore.java

```diff
@@ -182,9 +184,14 @@ public static String genSegmentFileName(String segmentId, String UUID) {
    * @param UUID      a UUID string used to construct the segment file name
    * @return segment file name
```

Comment:
add one more parameter `segmentMinMaxList`  in comment

Reply:
done